pyproject.toml

```diff
@@ -22,12 +22,15 @@ classifiers = [
 python = "^3.11"
 pydantic = "^1.8.2"
 requests = "^2.25.1"
+pytest = "^8.2.2"
+pytest-cov = "^5.0.0"
+pytest-mock = "^3.14.0"
```

Comment:
Make sure to add deps in the appropriate sections and don't add them multiple times. It seems to me that _this_ section should only contain deps required to _run_ the tool, not for running tests.

Also, the available groups are confusing to me: What is the difference between dev dependencies and the various groups like lint, test, types etc.?

Reply:
I will organize the dependencies in pyproject.toml correctly, ensuring no duplicates and categorizing them appropriately.